### PR TITLE
OAI-PMH: Improve labelling

### PR DESCRIPTION
**Pull request by Pineirin**
* closes: #1092

Screenshot from 2022-10-03 09-55-33
Screenshot from 2022-10-03 09-55-52

### CODE REVIEW

**Review by jennur — APPROVED** [review]

File: invenio_rdm_records/administration/views/oai.py [hunk 1]
```diff
@@ -61,9 +61,23 @@ class OaiPmhEditView(AdminResourceEditView):
...
-        "search_pattern": {"order": 4, "text": "Search query"},
+        "name": {
+            "order": 2,
+            "text": "Set name",
+            "description": "A short human-readable string naming the set.",
+        },
+        "spec": {
+            "order": 3,
+            "text": "Set spec",
+            "description": "An identifier for the set,"
```
Comment (ppanero) [inline invenio_rdm_records/administration/views/oai.py:72]:
```suggestion
            "description": "An identifier for the set, "
```

File: invenio_rdm_records/administration/views/oai.py [hunk 2]
```diff
@@ -61,9 +61,23 @@ class OaiPmhEditView(AdminResourceEditView):
...
+            "which cannot be edited after the set is created.",
```
Comment (ppanero) [inline invenio_rdm_records/administration/views/oai.py:73]:
```suggestion
            "it cannot be edited after the set is created.",
```

File: invenio_rdm_records/administration/views/oai.py [hunk 3]
```diff
@@ -61,9 +61,23 @@ class OaiPmhEditView(AdminResourceEditView):
...
+        },
+        "search_pattern": {
+            "order": 4,
+            "text": "Search query",
+            "description": "See the support query "
```
Comment (ppanero) [inline invenio_rdm_records/administration/views/oai.py:78]:
```suggestion
            "description": "See the supported query "
```

File: invenio_rdm_records/administration/views/oai.py [hunk 4]
```diff
@@ -61,9 +61,23 @@ class OaiPmhEditView(AdminResourceEditView):
     list_view_name = "OAI-PMH"
 
     form_fields = {
-        "name": {"order": 2, "text": "Set name"},
-        "spec": {"order": 3, "text": "Set spec"},
-        "search_pattern": {"order": 4, "text": "Search query"},
+        "name": {
+            "order": 2,
```
Comment (ppanero) [inline invenio_rdm_records/administration/views/oai.py:65]: question without background knowledge: shouldn't it start by 1?

File: invenio_rdm_records/administration/views/oai.py [hunk 5]
```diff
@@ -61,9 +61,23 @@ class OaiPmhEditView(AdminResourceEditView):
...
+            "text": "Set name",
+            "description": "A short human-readable string naming the set.",
```
Comment (ppanero) [inline invenio_rdm_records/administration/views/oai.py:67]: Should they be i18n? maybe @kpsherva or @jennur knows?

```suggestion
            "description": _("A short human-readable string naming the set."),
```

File: invenio_rdm_records/administration/views/oai.py [hunk 6]
```diff
@@ -82,9 +96,23 @@ class OaiPmhCreateView(AdminResourceCreateView):
     list_view_name = "OAI-PMH"
 
     form_fields = {
-        "name": {"order": 1, "text": "Set name"},
-        "spec": {"order": 2, "text": "Set spec"},
-        "search_pattern": {"order": 3, "text": "Search query"},
+        "name": {
+            "order": 1,
```
Comment (ppanero) [inline invenio_rdm_records/administration/views/oai.py:100]: This is what leads me to think above should be `1`.

Values are fully repeated? is there a way to re-use?

File: invenio_rdm_records/administration/views/oai.py [hunk 7]
```diff
@@ -82,9 +96,23 @@ class OaiPmhCreateView(AdminResourceCreateView):
...
+            "text": "Set name",
+            "description": "A short human-readable string naming the set.",
+        },
+        "spec": {
+            "order": 2,
+            "text": "Set spec",
+            "description": "An identifier for the set,"
```
Comment (ppanero) [inline invenio_rdm_records/administration/views/oai.py:107]:
```suggestion
            "description": "An identifier for the set, "
```

File: invenio_rdm_records/administration/views/oai.py [hunk 8]
```diff
@@ -82,9 +96,23 @@ class OaiPmhCreateView(AdminResourceCreateView):
...
+            "which cannot be edited after the set is created.",
```
Comment (ppanero) [inline invenio_rdm_records/administration/views/oai.py:108]:
```suggestion
            "it cannot be edited after the set is created.",
```

File: invenio_rdm_records/administration/views/oai.py [hunk 9]
```diff
@@ -82,9 +96,23 @@ class OaiPmhCreateView(AdminResourceCreateView):
...
+        },
+        "search_pattern": {
+            "order": 3,
+            "text": "Search query",
+            "description": "See the support query "
```
Comment (ppanero) [inline invenio_rdm_records/administration/views/oai.py:113]:
```suggestion
            "description": "See the supported query "
```

File: invenio_rdm_records/administration/views/oai.py (hunk 6 above)
Comment (kpsherva) [reply invenio_rdm_records/administration/views/oai.py:100]: for now, no, however we want to create a way to do that in the future -  for this release it was nice to have

**Pull request merged**